Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node
+ARG SERVICE_NAME
+
+# These steps ensure that npm install is only run when package.json changes
+RUN mkdir -p /usr/src/$SERVICE_NAME
+ADD ./package.json /usr/src/$SERVICE_NAME/package.json
+WORKDIR /usr/src/$SERVICE_NAME
+RUN npm install --production
+
+ADD . /usr/src/$SERVICE_NAME
```

Comment:
That's true. Unless we want to make it standard to include Dockerfiles with every Exoservice. Since each service will exist on some "service store" eventually, it might be useful to be transparent about what those images look like.